Guide To Contribute To Open Source.md

```diff
@@ -0,0 +1,66 @@
+# Beginners Guide To Contribute To Open Source
+What is ************Open Source************: Open source is anything that is created by person which could be a "Software" Or "Code" that is publicly accessible to users like ****you****. Doing this will allow you to "Modify" "Alter" "Change" The way you see fit. 
+
+What is the point of ************Open Source************: This allows users to take peices of that "Software" or "Code" and change it for their own liking, which then if the main creators believes that the "Software" Or "Code" is a greater than the oringal creation then it can be integrate into a bigger project or changed from the orginal creation.    
+
+
+
+
+## Steps on how to contribute to a open source Repositories 
+
+1. When wanting to contribute to open source you will need to fork a repository. A fork is how you copy and paste what ever files that are accessible to you. To fork you will need to goto the bit project and click the fork on the top right of the page. Once you do that it will copy the files to your repository
+
+![Image To Show Fork](https://docs.github.com/assets/images/help/repository/fork_button.jpg)
+
+2. You will then need to click on your profile link on the top right of the page and click my Repository once there you’ll see a copy version of bit project. 
+
+![Image for profile](https://i.gyazo.com/96289483c4166329dc522d424b13dd15.gif)
+
+3. Once in your own Repository you can then edit, download, and upload anything you want for your own liking.
+
+![Image for files](https://i.gyazo.com/b0f59f2a8b2794ec8b763dccfa949d45.gif)
+
+4. Once you feel that you have edited a file to your liking you would scroll down to "Commit Changes" And you would have two options, one "Commit directly to the **master** branch." Or "Create a new branch for this commit and start a pull request." Sounds difficult right? Well its not, What it means is you can either save all your edits into the fork or (copy) version of the repo you created. Or create a new branch and start a pull request which tell others about changes you've pushed to a branch in a repository on GitHub. But from our suggestions if you're working on multiple files it would be best to commit directly into your own fork before creating a pull request.
```

Comment:
this is only for markdown files, not code. you need to make that clear. just to reiterate we need to encourage people to clone + make changes locally instead of on github.com